backend/conferences/types.py

```diff
@@ -8,6 +8,9 @@
 from schedule.types import Room, ScheduleItem
 from submissions.types import Submission, SubmissionType
 
+if False:
```

Comment:
why do we need this? if it is for the type checking we can use TYPE_CHECKING: https://mypy.readthedocs.io/en/latest/common_issues.html#import-cycles

Reply:
fixed!